macros/parsers/parserCheckboxList.pl

```diff
@@ -532,8 +567,9 @@ sub CHECKS {
 	for my $i (0 .. $#{ $self->{orderedChoices} }) {
 		my $value = $self->{values}[$i];
 		my $tag   = $self->{orderedChoices}[$i];
-		$value = '%' . $value                                   if (grep { $i == $_ } @{ $self->{checkedI} });
-		$tag   = $self->labelFormat($self->{labels}[$i]) . $tag if $self->{displayLabels};
+		$value = '%' . $value if (grep { $i == $_ } @{ $self->{checkedI} });
+		$tag   = $self->labelFormat($self->{labels}[$i]) . $tag
+			if $self->{displayLabels};
```

Comment:
If lines 568-572 were changed to
```perl
		my $value = $self->{values}[$i];
		$value = '%' . $value if (grep { $i == $_ } @{ $self->{checkedI} });

		my $tag = $self->{orderedChoices}[$i];
		$tag = $self->labelFormat($self->{labels}[$i]) . $tag if $self->{displayLabels};

```
It would be better organized, and the empty lines improve readability as well as prevent perltidy from aligning things badly.